examples/blinky.rs

```diff
@@ -0,0 +1,94 @@
+#![no_std]
+#![no_main]
+#![feature(asm)]
+
+extern crate esp32_hal as hal;
+extern crate panic_halt;
+extern crate xtensa_lx6_rt;
+
+use hal::prelude::*;
+
+/// The default clock source is the onboard crystal
+/// In most cases 40mhz (but can be as low as 2mhz depending on the board)
+const CORE_HZ: u32 = 40_000_000;
```

Comment:
I don't intend to change this but to keep this as a side note, in the PR #7 this can be calculated at run time depending on the clock source used.

Reply:
Thanks for pointing that out. I'm fine with waiting until that PR has merged and updating the example if that's preferable.

Reply:
Maybe merge this PR first because #7 has been superseded by #11